### DISCUSSION TITLE
k7sfunc是否考虑更新rife引擎到4.20版本

### DISCUSSION THREAD

**Discussion by o5-null** [hooke007/mpv_PlayKit#489] [mpv 通用]
如题

**Comment by hooke007** [comment]
发版前同步最后一版，大概率不会是20

**Reply by o5-null** [reply]
rife版本貌似是版本越高性能需求越低

**Reply by hooke007** [reply]
你说的越高越低是指哪个版本开始

**Reply by hooke007** [reply]
4.0-4.6 缓步提高性能要求
4.7-4.12 暴涨
有lite版本开始波浪变动

**Reply by o5-null** [reply]
不同显示大小为什么会导致性能占用不同
rife不是在插帧前预缩放了视频吗
就算全屏显示也应该是仅使用双线性缩放

**Reply by hooke007** [reply]
> 不同显示大小为什么会导致性能占用不同

不同的渲染尺寸当然占用不同，我不是很明白你表达的意思

**Reply by o5-null** [reply]
没事了
缩放算法性能需求比我想得要大
